paper.md

```diff
@@ -0,0 +1,160 @@
+date: 01 October 2023
+bibliography: paper.bib
+---
+
+
+# Summary
+
+[`Libtrixi`](https://github.com/trixi-framework/libtrixi) [@schlottkelakemper2023libtrixi]
+is an interface library for [Trixi.jl](https://github.com/trixi-framework/Trixi.jl)
+[@schlottkelakemper2020trixi; @schlottkelakemper2021purely], a Julia package for adaptive,
+high-order numerical simulation of conservation laws. Example applications include
+high-speed flows with complex shock interactions, astrophysical simulations with
+self-gravity, shallow water problems for flood forecasting, and weather or climate
+prediction. `Libtrixi` provides an application programming interface (API) to Trixi.jl with
+C and Fortran bindings, which allows users to control the entire simulation process from
+their own main program. This includes setting up a simulation, advancing the numerical
+solution in time, and retrieving simulation data for further analysis. The main program may
+either be written in C/C++/Fortran or use any other language that can directly interact with
+the C/Fortran interface of `libtrixi`.
+
+With `libtrixi` it is possible to use the modern, high-order discretization schemes
+implemented in Trixi.jl within existing frameworks or legacy applications. Since Julia has
+similar high-performance properties as C/C++/Fortran, this approach facilitates the use of
+the rapid prototyping capabilities of Julia for developing new mathematical models or
+numerical methods, while continuing to use established codes for the overall process
+control. Both `libtrixi` and Trixi.jl are available under the MIT license.
+
+
+# Statement of need
+
+Numerical simulations of conservation laws are used to accurately predict many naturally
+occurring processes in various areas of physics, such as fluid flow, astrophysics, earth
+systems, or weather and climate modeling. These phenomena characteristically exhibit a broad
+range of spatial and temporal length scales, making it necessary to use finely resolved
+computational grids. Therefore, high-performance computing (HPC) techniques are required
+to render the numerical solution feasible on large-scale compute systems.
+
+Consequently, many simulation tools are written in traditional HPC languages such as C, C++,
+or Fortran, e,g., deal.II [@dealii2019design] or PETSc [@petsc-efficient]. These languages
+offer high computational performance, but often at the cost of being complex to learn and
+maintain. On the other hand, languages like Python, which are more amenable for rapid
+prototyping or less experienced users, are usually not fast enought to use without
+specialized packages that utilize kernels written in a compiled language. For example,
+Python's well-known NumPy library [@2020NumPy-Array] has its performance-critical code
+implemented in C.
+
+The Julia programming language [@Bezanson_Julia_A_fresh_2017] aims to provide a new approach
+to scientific computing. It strives to solve the two-language problem by providing an
```

Comment:
Is it clear that "two-language problem" is a fixed term which exactly refers to the problem just described in the preceding paragraph?

Reply:
You're right. It is not clear enough, and I hope it is clearer with 724588c.